### DISCUSSION TITLE
Support enterprise version

### DISCUSSION THREAD

**Discussion by neyugnluan** [quartznet/quartznet#1009] [Q&A] [answered]
I'm looking for this package a long time ago glad to see you are release a new version recently
Do you have any plan to make this as a purchase package or any way to support you?
I have a problem related to the xamarin android which reported issue.

**Comment by lahma** [comment, marked as answer]
If you find Quartz beneficial to you or even creating revenue, you can always consider sponsoring me via [GitHub sponsors](https://github.com/sponsors/lahma). All pull requests to fix issues are also very welcome that might be a easy way to both "pay back" and get your use cases to work.
